app/scripts/directives/istagSelect.js

```diff
@@ -21,31 +25,102 @@ angular.module("openshiftConsole")
       restrict: 'E',
       scope: {
         istag: '=model',
-        selectDisabled: '='
+        selectDisabled: '=',
+        includeOpenshift: '=',
```

Comment:
It's a mouthful, but maybe `includeSharedResourceNamespce`? It's actually customizable (see #13).